apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/GoogleLogger.cs

```diff
@@ -84,11 +86,11 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
                 Resource = _globalResource,
                 LogName = _logName,
                 Severity = logLevel.ToLogSeverity(),
-                Timestamp = Timestamp.FromDateTime(DateTime.UtcNow),
+                Timestamp = Timestamp.FromDateTime(_clock.GetCurrentDateTimeUtc()),
                 TextPayload = message,
             };
 
-            _consumer.Receive(new LogEntry[] { entry });
+            _consumer.Receive(new [] { entry });
```

Comment:
Nit: remove the space, so it's `new[] { entry }`. At least, that's how I'd always format it  :)